indico/modules/events/registration/client/js/form/fields/MultiChoiceInput.jsx

```diff
@@ -10,71 +10,101 @@ import PropTypes from 'prop-types';
 import React, {useState} from 'react';
 import {Field} from 'react-final-form';
 import {useSelector} from 'react-redux';
+import {Checkbox, Dropdown, Label} from 'semantic-ui-react';
 
 import {FinalCheckbox, FinalField} from 'indico/react/forms';
 import {Translate} from 'indico/react/i18n';
 
 import {getCurrency} from '../../form_setup/selectors';
 
 import {Choices, choiceShape} from './ChoicesSetup';
+import {PlacesLeft} from './PlacesLeftLabel';
 
 import '../../../styles/regform.module.scss';
+import './table.module.scss';
 
 export default function MultiChoiceInput({htmlName, disabled, choices, withExtraSlots}) {
+  // TODO: billable/price
```

Comment:
I think you already added this ;) Maybe this comment was accidentally brought back during a rebase?